src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
+      transition-duration: $duration;
```

Comment:
The variable $duration is not defined in this scope. Please ensure that the variable is defined, or use a specific duration like '0.5s'.